### PR TITLE
HomeOwner Variations and Bug fixes

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#14087 & /bcgov/entity#14096

*Description of changes:*
* updates to support the scenarios described in 14096 (Group removal scenarios)
* fixes bug outlined in 14087 (Unsaved changes bug)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/newMhrRegistration/HomeOwners.vue [hunk 1]
```diff
@@ -306,7 +316,18 @@ export default defineComponent({
...
+      hasSingleInvalidGroup: computed(() => {
+        const group = getTransferOrRegistrationHomeOwnerGroups().filter(group => group.action !== ActionTypes.REMOVED)
+        return !!group[0]?.interestNumerator && !!group[0]?.interestDenominator
+      }),
+      showHideOwnersBtn: computed(() => {
+        return props.isMhrTransfer &&
+          localState.hasRemovedOwners &&
+          (!showGroups || localState.hasRemovedAllOwners) &&
+          !localState.hasMultipleAddedGroups &&
+          !localState.hasSingleInvalidGroup
```
Comment (cameron-eyds) [inline ppr-ui/src/views/newMhrRegistration/HomeOwners.vue:330]: Just a bit of cleanup, pulling out the larger conditional from the template.

**Review by ChasBoucher — APPROVED** [review]
Looks good

**Review by dimak1 — APPROVED** [review]
Approved (but it's the same version as in main, you can sync and bump up the version).

**Comment by cameron-eyds** [comment]
> Approved (but it's the same version as in main, you can sync and bump up the version).

Thank you, will update!

**Review by owillborn — COMMENTED** [review]
looks solid

**Pull request merged**